app/serializers/billing-detail.js

```diff
@@ -0,0 +1,8 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  extract: function(store, type, payload, id, requestType) {
+    this.__requestType = requestType; // used by `extractMeta`
```

Comment:
FYI @blakepettersson - Since we are overloading ember-data-hal-9000's serializer the requestType needs to be cached.  Explanation can be found here:  https://github.com/201-created/ember-data-hal-9000/blob/master/addon/serializer.js#L37